core/src/main/java/org/wildfly/extras/creaper/core/online/OnlineManagementClientImpl.java

```diff
@@ -149,7 +149,8 @@ public void apply(Iterable<OnlineCommand> commands) throws CommandFailedExceptio
     public ModelNodeResult execute(ModelNode operation) throws IOException {
         checkClosed();
         operation = adjustOperationForDomain.adjust(operation);
-        log.debugf("Executing operation %s", ModelNodeOperationToCliString.convert(operation));
+        log.debugf("Executing operation %s\nJSON format:\n%s", ModelNodeOperationToCliString.convert(operation),
+                operation.toJSONString(false));
```

Comment:
Thanks for noticing. I have overlooked `operation.getOperation()` returns `ModelNode`.

Regarding newlines. I was considering both options. I have intentionally decided for this option, because of selecting JSON string. It is ensured than no unintentional character will be copied.